infrastructure/terraform/per_account/mgmt/modules/route53/main.tf

```diff
@@ -37,27 +37,27 @@ module "zones" {
 
 }
 
-resource "aws_route53_record" "prod_zone" {
-  zone_id = module.zones.route53_zone_zone_id["cpm.national.nhs.uk"]
-  name    = "api.cpm.national.nhs.uk"
-  records = [
-    "ns-453.awsdns-56.com.",
-    "ns-980.awsdns-58.net.",
-    "ns-1983.awsdns-55.co.uk.",
-    "ns-1103.awsdns-09.org."
-  ]
-  ttl  = 300
-  type = "NS"
-}
+# resource "aws_route53_record" "prod_zone" {
+#   zone_id = module.zones.route53_zone_zone_id["cpm.national.nhs.uk"]
+#   name    = "api.cpm.national.nhs.uk"
+#   records = [
+#     "ns-453.awsdns-56.com.",
+#     "ns-980.awsdns-58.net.",
+#     "ns-1983.awsdns-55.co.uk.",
+#     "ns-1103.awsdns-09.org."
+#   ]
+#   ttl  = 300
+#   type = "NS"
+# }
```

Comment:
How come we don't need a prod zone? And do we need the commented out code still? Or will we be adding it back at a later date so makes sense to keep the work done already?